script.js

```diff
@@ -1,4 +1,151 @@
+// there are 4 mode .
+// 1) player 1 roll . 2) player 1 pick to make the highest number.
+// 3) player 2 roll. 4) player 2 pick to make the highest number.
+var mode = 'player 1 roll dice';
+
+// store player 1 highest number
+var playerOneHighestNum = 0;
```

Comment:
Please be consistent whether you use "one" or "1" in your var names. Would recommend "1" because it's more clear and concise.